docs/source/upcoming_release_notes/1304-chin-rtds.rst

```diff
@@ -0,0 +1,30 @@
+1304 chin-rtds
```

Comment:
like the chin on my face?

Reply:
wait till you meet the mirrors with a tail.